hellodagger/src/main/java/com/airbnb/mvrx/helloDagger/Extensions.kt

```diff
@@ -0,0 +1,11 @@
+package com.airbnb.mvrx.helloDagger
+
+import androidx.annotation.IdRes
+import androidx.fragment.app.Fragment
+import com.airbnb.mvrx.helloDagger.base.MvRxActivity
+
+// Useful for getting access to a fragment in tests
+fun <A: Fragment> MvRxActivity.findFragmentById(@IdRes id: Int): A {
```

Comment:
Can you move this to your test class then?

Reply:
For some reason Android Studio says that the `supportFragmentManager.findFragmentById()` method is [inaccessible](https://user-images.githubusercontent.com/24315306/66856438-d5125680-efa2-11e9-8d31-c58c05cfa100.png) when I place this file is not in the main source set. It compiles and the test runs just fine though, so I think this could be an IDE bug. I have moved it to the `androidTest` source set now.